workflow/Snakefile

```diff
@@ -1,7 +1,7 @@
 import os
 import pandas as pd
 from snakemake.utils import min_version
-min_version("6.0")
+min_version("7.0")
```

Comment:
Oh if we're doing this, can you modify the abc-env.yml file to require snakemake version>=7?

Reply:
I think we should discuss the pros and cons of integrating resource requirement in the snake file. If we do want to do that, we should upgrade to snakemake 7.

Reply:
I feel like it doesn't hurt to require snakemake >= 7.0 so that we have the flexibility in the future to utilize resource requirements.

But i'll leave it to you if you wish to make the change in the abc codebase. I think if you do change this though, you should also change the abc_env.yml file